raiden-dapp/src/views/DisclaimerRoute.vue

```diff
@@ -1,10 +1,52 @@
 <template>
   <div data-cy="disclaimer" class="disclaimer">
     <div class="disclaimer__content">
-      <div class="disclaimer__content__paragraphs font-weight-light">
-        <p v-for="(paragraph, index) in $t('disclaimer.paragraphs')" :key="index">
-          {{ paragraph }}
-        </p>
+      <div class="disclaimer__content__text">
+        <span class="disclaimer__content__text__header">
+          {{ $t('disclaimer.user-info.header') }}
+        </span>
+        <span class="disclaimer__content__text__body">{{ $t('disclaimer.user-info.body') }}</span>
+        <div>
```

Comment:
What is the outside `<div>` for? :thinking:
If there is something surrounded needed, maybe a paragraph would suites better.

Reply:
The `div` was added because Vue does not allow me to have two `v-for` at the same "level". The console outputs: `Duplicate keys detected: '0'. This may cause an update error.`. I think it seems more correct to wrap an `<ul>` in a div rather than a paragraph?

Reply:
Please take a look [here](https://github.com/raiden-network/light-client/pull/2531#discussion_r566718964)

Reply:
👍🏼

Reply:
Done!